services/notify/message-bus/index.external.integration.test.js

```diff
@@ -2,7 +2,7 @@
 
 const createNotifyService = require('./index');
 
-describe('Notify service', () => {
+describe.skip('Notify service', () => {
```

Comment:
Is there a reason we're skipping tests rather than deleting them if they've become unnecessary due to removing functionality?